### PR TITLE
Script to create classes and items

### PR DESCRIPTION
**Pull request by SergiiTao**
Implemented a script to create classes and items

### CODE REVIEW

File: models/ItemTreeGenerator.php [hunk 1]
```diff
@@ -0,0 +1,257 @@
...
+        return [
+            'prefix' => 'h',
+            'longPrefix' => 'help',
+            'description' => 'Prints a help statement'
+        ];
+    }
+
+    protected function provideDescription(): string
+    {
+        return 'Tool to generate items';
```
Comment (ktsirkou) [inline models/ItemTreeGenerator.php:101]: Better to enhance the description to cover the class hierarchy structure generation

File: README.md [hunk 2]
```diff
@@ -2,3 +2,29 @@ extension-tao-devtools
...
 
 extension that gather develepment tools for TAO
+
+# Scripts
+## Item generator
+
+Generates given number of items and classes. 
+
+### Run
+```shell script
```
Comment (wazelin) [inline README.md:12]:
```suggestion
```shell
```

File: models/ItemTreeGenerator.php [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+                'prefix'       => 'c',
+                'required'     => false,
+                'description'  => 'Number of classes in class. Can be int or range. Example: 5, 1-5\',',
+                'defaultValue' => '2'
+            ],
+            self::OPTION_NESTING_LEVEL => [
+                'longPrefix'   => 'nesting-level',
+                'prefix'       => 'n',
+                'required'     => false,
+                'description'  => 'Nesting level. Can be int or range. Example: 5, 1-5\',',
```
Comment (wazelin) [inline models/ItemTreeGenerator.php:70]:
```suggestion
                'description'  => 'Nesting level. Can be int or range. Example: 5, 1-5',
```

File: models/ItemTreeGenerator.php [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+                'prefix'       => 'i',
+                'required'     => false,
+                'description'  => 'Number of items in class. Can be int or range. Example: 5, 1-5',
+                'defaultValue' => '2'
+            ],
+            self::OPTION_CLASS_COUNT   => [
+                'longPrefix'   => 'class-count',
+                'prefix'       => 'c',
+                'required'     => false,
+                'description'  => 'Number of classes in class. Can be int or range. Example: 5, 1-5\',',
```
Comment (wazelin) [inline models/ItemTreeGenerator.php:63]:
```suggestion
                'description'  => 'Number of classes in class. Can be int or range. Example: 5, 1-5',
```

File: models/ItemTreeGenerator.php [hunk 5]
```diff
@@ -0,0 +1,259 @@
...
+                'prefix'       => 'n',
+                'required'     => false,
+                'description'  => 'Nesting level. Can be int or range. Example: 5, 1-5\',',
+                'defaultValue' => '3'
+            ],
+            self::OPTION_OWN_ROOT      => [
+                'longPrefix'   => self::OPTION_OWN_ROOT,
+                'prefix'       => 'r',
+                'required'     => false,
+                'description'  => 'All classes create under own root',
```
Comment (wazelin) [inline models/ItemTreeGenerator.php:77]:
```suggestion
                'description'  => 'Create each class under its own individual root',
```

File: README.md (hunk 2 above)
Comment (SergiiTao) [reply README.md:12]: why?

Comment (wazelin) [reply README.md:12]: https://github.com/github/linguist/blob/master/lib/linguist/languages.yml.
There's no such a thing as `shell script`.

**Pull request merged**